### PR TITLE
update install link for Linux/Mac

### PR DESCRIPTION
**Pull request by adam3smith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Tesseract OCR is used for the text recognition itself.
 
 - Tesseract OCR is installed
   - for Windows see https://github.com/UB-Mannheim/tesseract/wiki
-  - for Linux, Mac see https://github.com/tesseract-ocr/tesseract/wiki
+  - for Linux, Mac see https://github.com/tesseract-ocr/tessdoc/blob/master/Installation.md
```
Comment (stweil) [inline README.md:19]:
```suggestion
  - for Linux, Mac see https://tesseract-ocr.github.io/tessdoc/Installation.html
```

**Review by stweil — CHANGES_REQUESTED** [review]
Thanks for adressing this. I suggest to link to the official documentation side.

**Review by stweil — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (adam3smith) [reply README.md]: Thanks -- I missed that this existed as a page

**Comment by stweil** [comment]
Thank you!

**Pull request merged**